deploy/01_deploy_signerV2-contract.js

```diff
@@ -15,6 +15,7 @@ const deployFunction = async ({ getNamedAccounts, deployments, ethers }) => {
 
   const deploySeedFunctionSignature =
     await seedFactoryInstance.interface.getSighash("deploySeed");
+  console.log(deploySeedFunctionSignature);
```

Comment:
Currently wanted because we need the function signature for verifying the contracts on Etherscan. Once I verifying the contracts more dynamic ee could remove this